third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -1,11 +1,20 @@
 #include "ElementwiseOpToLLVM.h"
+#include "triton/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.h"
+#include "triton/Conversion/TritonGPUToLLVM/PatternTritonGPUOpToLLVM.h"
 
 using namespace mlir;
 using namespace mlir::triton;
+
 using ::AMD::ConvertTritonGPUOpToLLVMPattern;
 using ::AMD::ConvertTritonGPUOpToLLVMPatternBase;
 using ::AMD::TritonGPUToLLVMTypeConverter;
+using mlir::triton::gpu::ElementwiseOpConversionBase;
+using mlir::triton::gpu::getElementType;
 using ::mlir::triton::gpu::getTotalElemsPerThread;
+using mlir::triton::gpu::MultipleOperandsRange;
+using mlir::triton::gpu::packI32;
+using mlir::triton::gpu::reorderValues;
+using mlir::triton::gpu::unpackI32;
```

Comment:
why do we need that? We have using `using namespace mlir::triton;` at the top already